plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -73,19 +80,19 @@ def upload(repo, type_id, unit_key, metadata, file_path, conduit, config):
 
     # Dispatch to process the upload by type
     handlers = {
-        models.RPM.TYPE : _handle_package,
-        models.SRPM.TYPE : _handle_package,
-        models.PackageGroup.TYPE : _handle_group_category,
-        models.PackageCategory.TYPE : _handle_group_category,
-        models.Errata.TYPE : _handle_erratum,
-        models.YumMetadataFile.TYPE : _handle_yum_metadata_file,
+        models.RPM.TYPE: _handle_package,
+        models.SRPM.TYPE: _handle_package,
+        models.PackageGroup.TYPE: _handle_group_category,
+        models.PackageCategory.TYPE: _handle_group_category,
+        models.Errata.TYPE: _handle_erratum,
+        models.YumMetadataFile.TYPE: _handle_yum_metadata_file,
     }
```

Comment:
These changes too. Many of these are likely already fixed on master anyway, as I believe master does have flake8 enforcement.